Python/Desktop Time Tracker/Readme.md

```diff
@@ -0,0 +1,34 @@
+# Time Tracker
+## Description
+This script is written in python and can be used to track user time used on Desktop Applications in a pie chart format
+* The script is mainly built using [xdotool](https://www.freebsd.org/cgi/man.cgi?query=xdotool&apropos=0&sektion=1&manpath=FreeBSD+8.1-RELEASE+and+Ports&format=html)
+
+### Language
+- [X] Python
+
+### Installation
+```bash
+$ sudo apt-get install xdotool
+$ pip install subprocess
```

Comment:
Can you add a `requirements.txt` with all these modules, without having them install individually?

Reply:
Done